constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintAttribute.java

```diff
@@ -103,7 +103,7 @@ public void setStringValue(String value) {
      *
      * @return Typically 1 but 3 for colors.
      */
-    public int noOfInterpValues() {
+    public int numberOfInterpolationValues() {
```

Comment:
numberOfInterpolatedValues ?

Reply:
Yeah that sounds better, will fix 👍🏻

Reply:
@camaelon the comment above in the docs says `The number of interpolation values that need to be interpolated` - should that be left alone? I think that was why I renamed it that way originally.